src/callbacks_step/analysis_dg2d.jl

```diff
@@ -138,7 +140,9 @@ end
 
 function calc_error_norms(func, u, t, analyzer,
                           mesh::Union{StructuredMesh{2}, StructuredMeshView{2},
-                                      UnstructuredMesh2D, P4estMesh{2}, T8codeMesh{2}},
+                                      UnstructuredMesh2D, P4estMesh{2},
+                                      P4estMeshView{2},
```

Comment:
```suggestion
                                      UnstructuredMesh2D, 
                                      P4estMesh{2}, P4estMeshView{2},
```

Not sure if the formatter accepts this - but I like semantic formatting.

Reply:
Seems like the formater is OK with it.